docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
 
 # Installing Regression Games
 
-import Placeholder from '../partials/_placeholder.mdx';
+This page covers everything you need to know to add the Regression Games SDK to your Unity project and start exploring its features.
 
-<Placeholder />
\ No newline at end of file
+## Add the Regression Games SDK to Your Project
+
+Our core package is [publicly available on GitHub](https://github.com/Regression-Games/RGUnityBots).
+Open your Unity project and navigate to `Window` > `Package Manager`. From here, find the `Add package from git URL` and paste the following URL:
+
+```
+https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.27
+```
+
+:::info
+- This package utilizes TextMeshPro. If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+- This package installs the [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) dependency for serializing/deserializing Json message payloads.
+- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart.
```

Comment:
Reading this line as a developer, without seeing the actual pop-up, can make me a bit concerned. I might think "Is adding regression sdk gonna change my whole input set up!?" consider adding a sentence to placate that potential fear/ add little bit more context.

If the pop-up itself is self explanatory this might be less of an issue, but we can't assume folks will read docs while going along.

Reply:
I don't actually know what this pop-up looks like. @RG-nAmKcAz I believe you've seen the pop ups for the "new input system" - do you have thoughts?

Reply:
Personally as a Unity dev, I find that the popup is self explanatory.  It tells you to restart to enable the new input system reading native device states.

Reply:
This pops up for me all the time since I make a lot of fresh projects from Boss Room. The dialog does have a good amount of information on what will happen, but yes I think adding a quick comment would be good. I personally don't know how much this changes in the project and how easy it is to "undo" the operation. Maybe we can just put something like 

```suggestion
- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart (this will allow the new input system to be enabled).
```

Here is what it looks like: 
![image](https://github.com/user-attachments/assets/1e8f759f-9174-451c-acf1-88ae18c1b240)